docs/index.rst

```diff
@@ -0,0 +1,60 @@
+.. P4 Control Plane documentation master file, created by
+   sphinx-quickstart on Fri Jul 28 18:19:43 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+===========================
+P4 Control Plane User Guide
+===========================
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Setup
+
+   guides/dpdk-guide
+   guides/es2k-guide
+   guides/tofino-guide
+
+.. toctree::
+   :maxdepth: 1
+   :caption: ACC Build
+
+   guides/es2k/installing-acc-sdk
+   guides/es2k/defining-acc-environment
+   guides/es2k/building-acc-stratum-deps
+   guides/es2k/building-acc-p4cp
+
+.. toctree::
+   :maxdepth: 1
+   :caption: ES2K P4 Programs
+
+   guides/es2k/compiling-p4-programs
+   guides/es2k/deploying-p4-programs
+   guides/es2k/running-infrap4d
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Clients
+
+   clients/gnmi-ctl
+   clients/p4rt-ctl
+   clients/sgnmi_cli
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Security
+
+   guides/security-guide
+   guides/installing-tls-certificates
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Updates
+
+   updates/development-update-23.21
+   updates/development-update-22.50
+   updates/development-update-22.44
+   updates/changes-from-p4-ovs
```

Comment:
I am not familiar with Sphinx so every new document should be added here? And what does the maxdepth indicate here?